### ISSUE TITLE
Custom sort option for all tabular data

### ISSUE THREAD

**Issue by maneesha** [carpentries/amy#1512]
Anytime tabular data is displayed, the user should be able to select a column to sort by.

See also #1507 and #1063.

**Comment by maneesha** [comment]
@sheraaronhurt This is to make it easier on the admin user to scan data and find what they want - especially useful for larger data sets.  Also, is there anything else that you think would make it easier for the admin user to view tabular data?

Which of the following applies to when you want to see this fixed?
* Would like to see this fixed as soon as possible, in the next 1-2 development cycles (i.e., by September 2019)
* Would like to see this fixed later (by the end of 2019)
* Would like to see this fixed, but possibly as part of a larger project that needs more planning
* Not sure
* No longer relevant - OK to close

**Comment by sheraaronhurt** [comment]
@maneesha  thanks for tagging me on this. I Would like to see this fixed later (by the end of 2019)

**Comment by pbanaszkiewicz** [comment]
@maneesha @sheraaronhurt  I'd like you to decide if you want me to add DataTables, just as in here: https://amy.carpentries.org/requests/training_requests/  - this would enable you not only to sort by clicking columns, as it would allow changing number of elements per page, and searching within.

**Comment by maneesha** [comment]
Yes - I think having all tabular views look like the training request view would be great.

**Comment by sheraaronhurt** [comment]
I agree with Maneesha, that will be great!

**Comment by pbanaszkiewicz** [comment]
Not sure how it works on your computers, but All Training Requests does slow down a bit on mine (using Windows and Firefox) due to the number of requests that need to load in.

**Comment by maneesha** [comment]
It's also slow for me too.  I think that's OK.  Maybe if the interface allows, we could have an hourglass or progress bar that indicates this.  Other than that, it would be great to use this feature across all our tables.

**Comment by pbanaszkiewicz** [comment]
@maneesha can you please change the title to "Use DataTables in all tabular data views", as it's more general than just changing sorting options?

**Comment by sheraaronhurt** [comment]
MIne slows down too. I thought it was me!
